applications/CoSimulationApplication/python_scripts/solver_wrappers/cpp_ping_pong/ping_pong_io.py

```diff
@@ -0,0 +1,72 @@
+from __future__ import print_function, absolute_import, division  # makes KratosMultiphysics backward compatible with python 2.6 and 2.7
+
+# Importing the base class
+from KratosMultiphysics.CoSimulationApplication.base_classes.co_simulation_io import CoSimulationIO
+from pathlib import Path
+import os
+
+def Create(model, settings, solver_name):
+    return PingPongIO(model, settings, solver_name)
+
+class PingPongIO(CoSimulationIO):
+    """This is the IO wrapper for the PING-PONG example.
+    """
+    def ImportCouplingInterface(self, interface_config):
+        pass
+
+    def ExportCouplingInterface(self, interface_config):
+        pass
+
+    def ImportData(self, data_config):
+        data_type = data_config["type"]
+        if data_type == "coupling_interface_data":
+            interface_data = data_config["interface_data"]
+            data_name = self.solver_name+"_"+interface_data.name
+            data_file_name = "EMPIRE_datafield_" + data_name + ".dat"
```

Comment:
why don't you use the CoSim- EMPIRE_API directly?

Reply:
as I said .. I want to show something else can also be done and it need not use what is existing ! 

But ya EMPIRE_API also will do the job !